frontend/packages/network-attachment-definition-plugin/src/constants/index.ts

```diff
@@ -62,5 +67,5 @@ export type NetworkTypeParameter = {
   required?: boolean;
   type: string;
   hintText?: string;
-  values?: { [key: string]: string };
+  values?: { [key: string]: any };
```

Comment:
```suggestion
  values?: { [key: string]: string };
```
AFAIU it's for `ELEMENT_TYPES.DROPDOWN` , we should not use it for default values
my guess is that we have a `value` or and `initValue: string | boolean` that we are missing

Reply:
i revert and added initValue : any , as it can be everything

Reply:
Init values of an html input of type `text`, `textarea`, `select` and `checkbox` is `string | boolean`

Reply:
the type actually helps to understand that it's actually the initial value used in this input because the type match the inputs

Reply:
but I do like better to initiate the `value` field (that is missing from the declaration here but used later after converting the object to `any` )

Reply:
using `value` is better because we do not introduce new fields, and it's an auto-init without needting to write code.
(not sure it works, so you will need to check)